.github/workflows/pull-request.yml

```diff
@@ -3,73 +3,7 @@ name: Ballerina HTTP module build
 on: [pull_request]
 
 jobs:
-  ubuntu-build:
```

Comment:
The workflow name should be: `PR Build`

Reply:
Ack, will change

Reply:
Fixed via [a773422](https://github.com/ballerina-platform/module-ballerina-http/pull/1763/commits/a77342294fbc3308d953608063d9027ce3b21d6a)